### PR TITLE
Add enquiry form

### PR DESCRIPTION
**Pull request by WebDevScott13**
Added contact route
Added Contact Page and Form
Added basic form validation
Added flash message on submission

### CODE REVIEW

File: resources/js/Pages/Enquiries/Index.tsx
Comment (chrisvasey) [inline resources/js/Pages/Enquiries/Index.tsx:1]: Probs delete this as won't be used.
We have some other index pages to build later though!

**Review by chrisvasey — CHANGES_REQUESTED** [review]
Just a few small things to clean up

File: resources/js/Pages/Enquiries/Index.tsx
Comment (WebDevScott13) [reply resources/js/Pages/Enquiries/Index.tsx:1]: This has been deleted. WIll push changes shorlty

File: app/Http/Controllers/EnquiryController.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+   * Display a listing of the resource.
+   */
+  public function index(): Response
+  {
+    // return response('Mamba Enquiries!');
+    return Inertia::render('Enquiries/Index', [
+      //
+    ]);
+
+    // @todo need to add the route for the contact page here
```
[the remark this answers is not in the document]
Comment (WebDevScott13) [reply app/Http/Controllers/EnquiryController.php]: I have removed this code from teh file

**Pull request merged**